src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -88,10 +88,10 @@ def __call__(self, t, smoothed=True):
             else:
                 return pred_rv
 
-        elif t > self.locations[-1]:
-            if smoothed:
-                warn("`smoothed=True` is ignored for extrapolation.")
-            return self._predict_to_loc(t)
+        # else: t > self.locations[-1]:
+        if smoothed:
```

Comment:
Should this be
```
if smoothed and t > self.locations[-1]:
```
to retain the same behaviour? Clearly no tests are failing so I'm not sure, but according to codecov this is also not covered.

Reply:
The "if" statements before check for `t < self.locations[0]` and `self.locations[0] < t < self.locations[-1]`, so the check is not actually necessary. I just wrote it that way as first to make the code more readable, and thought now that the comment achieves the same thing.